pallets/ocex/src/tests.rs

```diff
@@ -651,7 +651,7 @@ fn collect_fees() {
 		let snapshot =
 			EnclaveSnapshot::<AccountId32, WithdrawalLimit, AssetsLimit, SnapshotAccLimit> {
 				snapshot_number: 1,
-				merkle_root: mmr_root,
+				snapshot_hash: mmr_root,
```

Comment:
Please update var name as well. Do we need var definition at all here? we can just remove `let mmr_root: H256 = H256::random();` and add `snapshot_hash: H256::random()`